app/assets/stylesheets/hyrax/_variables.scss

```diff
@@ -16,3 +16,9 @@ $marketing-text-shadow: 1px 1px 0 rgba(0, 0, 0, 0.1) !important;
 $footer-top-margin: 3 * $padding-small-vertical !default;
 $footer-dashboard-top-margin: 0 !default;
 $footer-height: 118px !default;
+
+// Brand colors
+$brand-danger: #d33a35;
```

Comment:
These are bootstrap variables, not hyrax variables, so I wonder if this is the right location.  If you put them here they may not be used in bootstrap.  Also we should set `!default` so the consumer could provide their own defaults.

Reply:
We need these variables set prior to this: https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/templates/hyrax.scss#L8 but the code you provided doesn't get included until https://github.com/samvera/hyrax/blob/master/lib/generators/hyrax/templates/hyrax.scss#L15

Reply:
Yes, I ran into that. Removing the default allowed the variables to override the bootstrap defaults without changing the load order. I can create a new SASS file and load it before bootstrap. Does that seem like a reasonable alternative?